Packs/CybleThreatIntel/Integrations/CybleThreatIntel/Cyble_Logo.svg

```diff
@@ -0,0 +1 @@
0,0-.1-1.88c-1.41-3.35-4.42-10.45-4.57-10.72.16,0,15.43,4.65,15.7,4.74l-2,1.1L20.35,41c-.63.35-.62.53,0,1q4.79,3.51,9.54,7l.64.45a30.65,30.65,0,0,0-1.14,5.45C29.21,56.39,29.35,59.41,29.34,60.68Z"/><path d="M44.87,26.71c.11-.12.19-.26.11-.39a.76.76,0,0,0-.34-.24,2.14,2.14,0,0,0-.42-.16c-.35-.11-.51.18-.51.49s.14.61.42.62a.91.91,0,0,0,.43-.1A1.28,1.28,0,0,0,44.87,26.71Z"/><path d="M45.26,27.09c-.38.18-1.43.85-1.6.95-1.6,1-3.22,1.93-4.8,2.94-1.88,1.21-3.72,2.49-5.6,3.72-.16.11-.43,0-.65,0a3.87,3.87,0,0,1,.25-.59c0-.09,5.22-3.68,6.53-4.63-.3-.19-5.16-2.72-5.64-2.9-.31-.11-.51.06-.53.44s-.1,1-.15,1.47c-.22,2-.45,4.07-.64,6.11a.7.7,0,0,0,.23.58c.3.19.69.23,1,.39,2.39,1.22,4.74,2.51,7.16,3.65a1.94,1.94,0,0,0,1.79.09,1.79,1.79,0,0,0,.79-1.26c.72-3.33,1.53-6.64,2.33-10A1.26,1.26,0,0,0,45.26,27.09Z"/><path d="M49.49,25c-.05-.19-.25-.32-.38-.46a7,7,0,0,0-.83-.7c-.43-.33-.87-.65-1.32-.94-.9-.57-1.79-1.14-2.72-1.65-1.44-.79-3-1.37-4.47-2a1.68,1.68,0,0,0-2,.32L35.5,21.67a4.44,4.44,0,0,0-.33.37c.94.32,1.83.61,2.71.93l5.33,2c.31.12,2,.6,2.16.65a4.89,4.89,0,0,0,2.54.17,9.92,9.92,0,0,0,1.32-.3A.44.44,0,0,0,49.49,25Z"/><path d="M32.94,23.07,31.69,34.71l-15-4.23v-.21Z"/><path d="M32.64,22.43c-5.69,2.48-11.19,5-16.75,7.42-.36,0-.4-.21-.31-.55.76-2.78,1.51-5.58,2.29-8.36a3.42,3.42,0,0,1,.62-1.37,3.39,3.39,0,0,1,3.83-.76c3.1,1.52,6.48,2.21,9.7,3.4Z"/><path d="M43.93,38.82c.21-.94.47-1.93.67-2.79.63-2.68,1.25-5.36,1.9-8,.06-.26.26-.66.43-.68a1.17,1.17,0,0,1,.85.34Q51.41,31.25,55,34.9a1.2,1.2,0,0,1,.3.73c0,1.15,0,3.53,0,3.53a17,17,0,0,0-2.56-.68,19.85,19.85,0,0,0-3.7-.27,32.92,32.92,0,0,0-3.71.29A8.39,8.39,0,0,0,43.93,38.82Z"/><path d="M20.93,41.45l10.68-5.88c0,.37.05.64.07.91.15,2.82.25,6.69.52,9a4,4,0,0,1-.66,2.55,5.71,5.71,0,0,1-.65.7,9.09,9.09,0,0,1-.8-.52L21.4,41.83C21.26,41.73,21.13,41.61,20.93,41.45Z"/><path d="M56,34.9l-7.57-7.61c-.2-.21-.55-.41-.29-.74a1.13,1.13,0,0,1,.68-.31,7.31,7.31,0,0,1,3.45.07,11.61,11.61,0,0,1,7.33,7.23,11.75,11.75,0,0,1,0,7.23c-.16.48-.64,1.67-.64,1.67s-.51-.38-.6-.47c-.71-.66-2.39-2.35-2.39-2.35Z"/><path d="M16.86,54.07,18.13,48c1.31.89,2.55,1.72,3.78,2.57a1.62,1.62,0,0,1,.34.47L28.4,60.6c.17.28.36.55.55.82l-.12.1Z"/><path d="M32.86,45.75c-.17-3.24-.34-6.89-.52-10.1.33.19.59.37.86.51l6.72,3.48a7,7,0,0,1,.63.37s-.54.31-.84.51c-.88.57-1.78,1.13-2.65,1.73l-.76.53c-.6.45-1.19.92-1.75,1.42S33.43,45.24,32.86,45.75Z"/><path d="M19.27,42,18,46.65,9.31,40.74l0-.19Z"/><path d="M33,21.75c-.3-.08-8.82-2.83-9.07-3s-.12-.3,0-.45a1.35,1.35,0,0,1,.47-.35,5.29,5.29,0,0,1,1.32-.31c.31,0,.62-.08.94-.11,1.06-.11,2.13-.12,3.2-.14a1.44,1.44,0,0,1,1.37,1C31.73,19.33,32.91,21.48,33,21.75Z"/><path d="M37.24,19C34.91,21.17,34,21.9,34,21.9c-.65-1.21-1.28-2.34-1.89-3.49-.17-.33.15-.73.66-.78a3,3,0,0,1,.78.07l2.49.4C36.54,18.2,37.85,18.45,37.24,19Z"/><path d="M39.7,24.44a.15.15,0,0,1,0,.09c0,.42-.76.57-1.06.59-.72.06-1.44.09-2.16.1s-2.09.28-2.74-.43a1.29,1.29,0,0,1-.22-1,7.41,7.41,0,0,1,.11-1c1.1-.52,2.14.12,3.19.51l1.75.66C38.78,24,39.58,24.17,39.7,24.44Z"/></svg>
```

Comment:
Logo should match the requirements (120x50 px PNG, transparent background, <10kB in Size)